### ISSUE TITLE
dsp is never defined in Chapter 5, among others

### ISSUE THREAD

**Issue by gummybirdy** [GeoDaCenter/OpenAirQ-toolkit#9]
The dsp variable, used widely in Chapter 5, is never set in the code, making it quite hard to follow. I was not able to discern exactly what the dsp object was meant to contain.
On a somewhat related note, I found the abbreviation of variable names in this chapter excessive, making it harder still for me to follow along.

**Comment by Makosak** [comment]
ah, this is really helpful! yes will need to update. 

from what I can tell, dsp is reference the individual sensor points (that would be loaded in as a spatial data point)

we could also cut down on the total # of libraries, as instead of sp and sf, could use the st to sp function in sf... 

also maybe link to other articles about each method at the beginning of each section

Assigned to Marynia or Isaac; TBD
